ballerina/modules/types.any/any.bal

```diff
@@ -99,9 +99,10 @@ isolated function getUrlSuffixFromValue(ValueType anyMessage) returns string {
         return "google.protobuf.Empty";
     } else if anyMessage is record {||} {
         return "google.protobuf.Empty";
-    } else {
+    } else if anyMessage is record {} {
         return externGetNameFromRecord(anyMessage);
     }
+    return "";
```

Comment:
Shall we put a warning log here and return it empty.

Reply:
We'll add it as a separate task.